src/Raven.Server/RavenServer.cs

```diff
@@ -445,6 +459,71 @@ private void ListenToNewTcpConnection(TcpListener listener)
             });
         }
 
+        private readonly Lazy<ApiKeyAuthenticator> _authenticator =
```

Comment:
Why do it like this? The authenticator isn't an expensive class to create, and it doesn't seems to be used anywhere.